Program.cs

```diff
@@ -136,15 +208,19 @@ private static IEnumerable<string> GetExistingPaths(IEnumerable<XmlNode> nodes,
                 .Select(node =>
                 {
                     Debug.Assert(node.Attributes != null, "node.Attributes != null");
-                    return node.Attributes["Include"].Value;
+                    string val = node.Attributes["Include"].Value;
+                    if (nodeName == "Folder") {
+                        val = val.TrimEnd('\\');
+                    }
+                    return val;
                 })
                 .ToArray();
         }
 
         private static XmlNode CreateItemNode(XmlNode projectNode, string nodeName, string path)
         {
             XmlDocument projectFile = projectNode.OwnerDocument;
-            Debug.Assert(projectFile != null, nameof(projectFile) + " != null");
+            Debug.Assert(projectFile != null, "projectFile != null");
```

Comment:
Why this change?

Reply:
Seems to be a newer C# feature that my version of the .NET SDK (through MSBuild in `C:\Windows\Microsoft.NET\Framework64\v4.0.30319\`) didn't support. Seemed to be the only thing that increased the version requirement, so I dropped it.